### PR TITLE
Antipodes coordinates

### PR DESCRIPTION
**Pull request by Mike-3**
Icon ok ?

### CODE REVIEW

File: lib/widgets/registry.dart [hunk 1]
```diff
@@ -1049,6 +1050,12 @@ class Registry {
         iconPath: 'assets/coordinates/icon_equilateral_triangle.png',
         searchStrings: SEARCHSTRING_COORDINATES + 'equilateral triangles gleichseitiges dreiecke'
       ),
+      GCWTool(
+          tool: Antipodes(),
+          i18nPrefix: 'coords_antipodes',
+          iconPath: 'assets/coordinates/icon_antipodes.png',
+          searchStrings: SEARCHSTRING_COORDINATES + 'gegenüber opposide erde earth punkt point'
```
Comment (S-Man42) [inline lib/widgets/registry.dart:1057]: please:
- gegenüber -> gegenueberliegende
- opposide -> oppositeside
- remove rest :) Could be confusing. All coordinates work with earth or points
- add "antipodes antipoden gegenpunkte"

File: assets/i18n/en.json [hunk 2]
```diff
@@ -909,6 +909,9 @@
   "coords_resection_coordc" : "Coordinate C",
   "coords_resection_angle12" : "Angle between A and B",
   "coords_resection_angle23" : "Angle between B and C",
+  "coords_antipodes_title" : "Antipodes",
+  "coords_antipodes_description" : "Calculates the point on the opposite side of the earth",
```
Comment (S-Man42) [inline assets/i18n/en.json:913]: instead of "earth", more correctly "ellipsoid (e.g. the earth)"

File: assets/i18n/de.json [hunk 3]
```diff
@@ -909,6 +909,9 @@
   "coords_resection_coordc" : "Koordinate C",
   "coords_resection_angle12" : "Winkel zwischen A und B",
   "coords_resection_angle23" : "Winkel zwischen B und C",
+  "coords_antipodes_title" : "Antipode",
+  "coords_antipodes_description" : "Berechnet den Punkt auf der gegenüberliegenden Seite der Erde",
```
Comment (S-Man42) [inline assets/i18n/de.json:913]: instead of "Erde" more correctly "Ellipsoid (z.B. die Erde)"

File: lib/widgets/tools/coords/antipodes.dart [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        ),
+      ],
+    );
+  }
+
+  _calculateOutput() {
+    _currentValues = [antipodes(_currentCoords)];
+
+    _currentMapPoints = [
+      MapPoint(
```
Comment (S-Man42) [inline lib/widgets/tools/coords/antipodes.dart:69]: Please add the start point as well

File: lib/widgets/selector_lists/coords_selection.dart [hunk 5]
```diff
@@ -43,6 +44,7 @@ class CoordsSelection extends GCWSelection {
           className(Intersection()),
           className(Resection()),
           className(EquilateralTriangle()),
+          className(Antipodes()),
```
Comment (S-Man42) [inline lib/widgets/selector_lists/coords_selection.dart:47]: IMHO you could move this function higher. Maybe after Centerpoints of three Coords before Cross Bearing

File: lib/widgets/tools/coords/antipodes.dart (hunk 4 above)
Comment (Mike-3) [reply lib/widgets/tools/coords/antipodes.dart:69]: ok, but that is a very long distance

**Pull request merged**